src/modules/layouts/utils.ts

```
@@ -36,4 +36,457 @@ export function compareValues(
+ * Get group description for display
+ */
+export const getGroupDescription = (
+  groupingValue: GroupingValue | null,
+  columnType: ColumnType | null
+): string => {
+  if (!groupingValue || !columnType) return "";
+  
+  switch (columnType) {
+    case "numeric":
+      const numericValue = groupingValue as NumericBy;
+      return `${numericValue.from} to ${numericValue.to}`;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)